src/textacy/extract/triples.py

```diff
@@ -249,53 +256,47 @@ def direct_quotations(doc: Doc) -> Iterable[DQTriple]:
                 # if tok.pos in {NOUN, PROPN}
                 if not (tok.is_punct or tok.is_stop)
             )
-            # TODO: require closing punctuation before the quotation mark?
-            # content[-2].is_punct is False
         ):
             continue
-        # get window of adjacent/overlapping sentences
-        window_sents = (
-            sent
-            for sent in doc.sents
-            # these boundary cases are a subtle bit of work...
-            if (
-                (sent.start < qtok_start_idx and sent.end >= qtok_start_idx - 1)
-                or (sent.start <= qtok_end_idx + 1 and sent.end > qtok_end_idx)
-            )
-        )
+
+        triple = None
+        for n, window_sents in enumerate([
+            windower(qtok_start_idx, qtok_end_idx, doc, True), 
+            windower(qtok_start_idx, qtok_end_idx, doc)
```

Comment:
is it possible for these two approaches to ever produce identical windows? does that matter?